index.js

```diff
@@ -70,6 +73,7 @@ RedirectableRequest.prototype.write = function (data, encoding, callback) {
     this.emit("error", new Error("Request body larger than maxBodyLength limit"));
     this.abort();
   }
+  return callback;
```

Comment:
Why?

Reply:
ES-Lint, obligated to return a value for the function since I'm returning the callback at line 65.

Reply:
So why are we returning on line 65?

Reply:
To fix the issue of the callback not being called on empty buffers. Although, I could make a call to the ```.end()```method.

Reply:
Can't you just call `callback` and not return?